dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/channel/HttpDispatcherLink.java

```diff
@@ -1208,13 +1207,13 @@ public boolean isHTTP2UpgradeRequest(Map<String, String> headers, boolean checkE
      * @return false if some error occurred while servicing the upgrade request
      */
     @Override
-    public boolean handleHTTP2UpgradeRequest(Map<String, String> headers) {
+    public boolean handleHTTP2UpgradeRequest(Map<String, String> http2Settings) {
```

Comment:
I'm not sure http2Settings is as descriptive here. While the idea is to get the settings header, in particular HTTP2-Settings, the map I believe remains inherently a map of headers and not of settings.

Reply:
Ok I see there is a change on the method creating this map so that just the settings related headers are added when present, so I changing this name here makes more sense now. The Map is still a key value pair representing headers, albeit specifically now to the Upgrade/Connection. Since at this point it should have already confirmed whether it is H2 or not, I don't have strong feelings towards this name change, so I'll disregard my first message. Maybe something like http2ConnectionSettings might make sense, but again no strong objection to your proposed name change.